Assignment2/toasterdoom-fromRDFtoJSON-LD.jsonld

```diff
@@ -0,0 +1,34 @@
+{
+    "@context": {
+        "base": "http://assignment2.org/#",
```

Comment:
you don't need to define this base prefix.

Reply:
Thanks for the feedback! It's fixed now